nmdc_runtime/api/endpoints/queries.py

```diff
@@ -62,7 +63,7 @@ def run_query(
     }
     ```
     """
-    if isinstance(query_cmd, DeleteCommand):
+    if isinstance(query_cmd, (DeleteCommand, UpdateCommand)):
         check_can_delete(user)
```

Comment:
I think there is an inconsistence between the name of the function being called here (i.e. `..._delete`) and the conditions for calling it.

Reply:
Good catch! We did not create a `check_can_update`, since it would apply the same rules as `check_can_delete`. I'll document this in the code.